src/supervisor.tex

```diff
@@ -2258,3 +2273,97 @@ \chapter{``Svnapot'' Standard Extension for NAPOT Translation Contiguity, Versio
   configurations result in page-faults exceptions, while invalid access
   types or accesses to invalid physical memory regions trigger page faults.
 \end{commentary}
+
+\chapter{``Svpbmt'' Standard Extension for Page-Based Memory Attributes, Version 0.1}
+\label{svpbmt}
+
+{\bf Warning! This draft specification is likely to change before being
+accepted as standard by the RISC-V Foundation.}
+
+In Sv39, Sv48, and Sv57, bits 62--61 of the page table entry indicate the use
+of page-based memory types that override the PMA(s) for the associated memory
+pages.  The encoding for the PBMT bits is captured in Table~\ref{pbmt}.
+
+\begin{table*}[h!]
+\begin{center}
+\begin{tabular}{|r|l|}
+\hline
+Value  & Page-Based Memory Attributes \\
+\hline
+0      & None \\
+1      & Non-cacheable, idempotent, weakly-ordered (RVWMO or RVTSO), main memory \\
+2      & Non-cacheable, non-idempotent, strongly-ordered (channel 0), I/O \\
```

Comment:
Where is "channel 0" explained? The Unprivileged ISA also has a couple of mentions of channels, but no real explanation of what they mean.

Reply:
Nevermind...found this in the Privileged ISA section 3.6.4 Memory-Ordering PMAs. (I was expecting it in the RVWMO material in the Unprivileged ISA.)